test/regression/fragment/input.py

```diff
@@ -0,0 +1,94 @@
+database(
+    thermoLibraries=["primaryThermoLibrary", "SulfurLibrary", "DFT_QCI_thermo"],
```

Comment:
Probably a nitpick but since you are only generating for all carbon fragment, the sulfur library is not needed

Reply:
Good catch, done